bundle/edu.gemini.itc.web/src/main/scala/edu/gemini/itc/web/ITCRequest.scala

```diff
@@ -41,7 +46,10 @@ sealed abstract class ITCRequest {
   def intParameter(name: String): Int = parameter(name).toInt
 
   /** Gets the named value as a double. */
-  def doubleParameter(name: String): Double = parameter(name).toDouble
+  def doubleParameter(name: String): Double = parameter(name).trim() match {
+    case ""   => 0.0
+    case s    => s.toDouble
+  }
```

Comment:
Yes, I need to revisit the validation of the handling of parameters here, but I think that's going to be a bigger chunk of work. Probably the int values should be handled the same, currently this code is sort of geared towards the needs of the HTML code which does define defaults for all ints but sends an empty string for the wavelength (double). I'll add the same handling for ints though. And validation is soon to come.. I hope.